local_tests/build-docker-python.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
+set -e
+
 # Save the current path
 CURRENT_PATH=$(pwd)
 
 # Build the extension
-ARCHITECTURE=amd64 VERSION=1 ./scripts/build_binary_and_layer_dockerized.sh
+ARCHITECTURE=amd64 VERSION=1 AGENT_VERSION=$AGENT_VERSION ./scripts/build_binary_and_layer_dockerized.sh
```

Comment:
maybe we can set a dummy AGENT_VERSION here as we're dealing with local testing (and remove the set -e) ?

Reply:
if agent version isn't specified in the when running `build-docker-python.sh`, `build_binary_and_layer_dockerized.sh` will not override the default agent version and the default will be used. How come we want to remove set -e?